### DISCUSSION TITLE
20230515

### DISCUSSION THREAD

**Discussion by tmiw** [ARSFI/paclink-open-source#35] [General]
This is a pre-production build intended to gauge current status of the project and determine what needs to be fixed. Brief testing indicates that packet (using Kenwood TM-D700A), AGW (using AGWPE and UZ7HO soundmodem), Telnet and Pactor channels work.

Contains (vs. [last pre-release version](https://github.com/ARSFI/paclink-open-source/releases/tag/nightly_20221111)):

* Fix bugs related to high DPI scaling on Windows.

Instructions:

1. Unzip paclink-open-source.zip somewhere on your hard drive.
2. Run setup.exe to install on your system. Paclink will be installed in the Start menu.
3. Open Paclink from the Start menu and reconfigure if required. (There is no import functionality from the existing Paclink at this time but if you used a pre-release version, configuration should be saved.)

Please leave comments in the release's discussion thread or email K6AQ@winlink.org with feedback. 👍

<hr /><em>This discussion was created from the release <a href='https://github.com/ARSFI/paclink-open-source/releases/tag/nightly_20230515'>20230515</a>.</em>

**Comment by Spirarementum** [comment]
Well-done :)
No more display scaling issues with this version.

Standing by to test Ten-Tec T588 rig remote radio control once you get to this.

Otherwise let me know if  can be of further testing help but be advise I will likely be at sea for a month starting mid May and might not get Internet access for a while after this.

Serge VA2SR@winlink.org
